docs/UserGuide.md

```diff
@@ -348,21 +349,21 @@ Lists all expenses in the finance tracker.
 
 #### 4.3.5 Find Expense: `find`
 
-Finds expenses that match any of the given parameters, and displays the list of matching expenses.
+Finds expenses that match any of the given search parameters, and displays the list of matching expenses.
 
 Format: (when on the [Expenses tab](#223-expenses-tab)) `find [t/TITLE_KEYPHRASE...] [a/AMOUNT] [d/DATE] [c/CATEGORY...] [af/AMOUNT_FROM] [at/AMOUNT_TO] [df/DATE_FROM] [dt/DATE_TO]`
```

Comment:
The message displayed in the application uses `t/KEYWORD` instead of `t/TITLE_KEYPHRASE`.